tests/js/tstdlib_imports.nim

```diff
@@ -64,4 +64,7 @@ import
   # works but shouldn't: segfaults
 
   # Modules for JS backend:
-  asyncjs, dom, jsconsole, jscore, jsffi
\ No newline at end of file
+  asyncjs, dom, jsconsole, jscore, jsffi
+
+  # Unlisted in lib.html:
+  std/[decls, compilesettings, with, wrapnils]
```

Comment:
this gives error; you can use this: https://gist.github.com/timotheecour/058320de3093737967b82d57d95ff09c

which has a single `std/[` at top-level, i feel it's cleaner

Reply:
Weirdly, you cannot `import std/unidecode`